modules/lib/core/README.md

```diff
@@ -0,0 +1,52 @@
+# Enonic XP Core TS types
+
+> TypeScript definitions that are shared between libraries and represent core data structures.
+
+## Install
+
+```bash
+npm i --save-dev @enonic-types/core
+```
+
+## Use
+
+> It is not necessary to include this dependency into your project, as all libraries export related core types,
+> e.g. Content library exports `Content` type.
+
+Add the corresponding types to your `tsconfig.json` file that is used for application's server-side TypeScript code.
+
+`tsconfig.json`
+
+```json
+{
+  "compilerOptions": {
+    "types": [
+      "@enonic-types/core"
+    ]
+  }
+}
+```
```

Comment:
My concern was that you must import that library to update/merge `XpXData`, unlike `XpLibraries` interface from the libs. But I guess, since it's empty and won't be used without Content, we can remove this.

Reply:
Yes, I think `XpXData` til be created globally as soon as *core* is imported (to be able to use `Content`)